### ISSUE TITLE
How to train a pretrained quantlize model by myself

### ISSUE THREAD

**Issue by Changkou** [tensorflow/models#4814]
I trained a mobile_v1 quantlize model with my own datasets, and the accuracy decayed for almost 10% in compare with the float model, all the other parameters are the same, so i want to know how to train a quantlize pretrained  model.
I know it was transfered from a float model, but i want to know the detail information like how many steps used to train a float model before tranfer to a quantlize model.
Thanks, and sorry for my poor english.

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
What is the top-level directory of the model you are using
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by Changkou** [comment]
I trained a quantlized mobile net v1 in slim-nets directory
The platform i use is Ubuntu 16.04
Tensorflow was installed by pip and the version is tensorflow-gpu==1.9.0rc0
Bazel version is 0.13.0
CUDA version is 8.0 cudnn version is 5.1.10 and i have another CUDA whose version is 9.0 and cudnn is 7.0 i remember
GPU is gtx 1080 8GB

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be considerably old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.
